contingency/contingency-api/src/main/java/com/powsybl/contingency/StaticVarCompensatorContingency.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.contingency;
+
+import com.powsybl.contingency.tasks.AbstractTrippingTask;
+import com.powsybl.contingency.tasks.StaticVarCompensatorTripping;
+
+import java.util.Objects;
+
+/**
+ * @author Teofil Calin BANC <teofil-calin.banc at rte-france.com>
+ */
+public class StaticVarCompensatorContingency implements ContingencyElement {
```

Comment:
Is it a good idea to refactor the code to introduce an InjectionContingencyElement to share code between all injections?

Reply:
Intermediate class has been added.